modules/gridmap/grid_map.cpp

```diff
@@ -572,11 +572,12 @@ bool GridMap::_octant_update(const OctantKey &p_key) {
 				NavigationServer3D::get_singleton()->region_set_map(region, get_world_3d()->get_navigation_map());
 				nm.region = region;
 
+#ifdef DEBUG_ENABLED
 				// add navigation debugmesh visual instances if debug is enabled
 				SceneTree *st = SceneTree::get_singleton();
 				if (st && st->is_debugging_navigation_hint()) {
 					if (!nm.navmesh_debug_instance.is_valid()) {
-						RID navmesh_debug_rid = navmesh->get_debug_mesh()->get_rid();
+						RID navmesh_debug_rid = navmesh->_get_debug_mesh()->get_rid();
```

Comment:
Unrelated, should be removed (you can open an issue about what that bug is because it's weird you keep changing it - but I suspect you might be working off a slightly outdated `master` branch if this breaks).

Reply:
This happens on all my rebases up to last night. I also have the same error on ming-llvm and osx's clang.